examples/triggers/build.xml

```diff
@@ -57,7 +57,12 @@
 		</jar>
 	</target>
 
+	<target name="install" depends="jar">
```

Comment:
Instructions in readme were telling to copy this jar manually to `conf/triggers`. There is no reason a user has to do that step manually if we can automate it.